### PR TITLE
Update training phase to reflect custom document(Train-ticket)

### PR DESCRIPTION
**Pull request by fakela**
Update training phase to reflect custom document: add new images, links and new information to reflect new document

### CODE REVIEW

**Comment by fharper** [comment]
Should be use other type of documents than off-the-shelf API ones for the API Builder examples? Rent receipt is still a receipt no?

**Comment by fakela** [comment]
> Should be use other type of documents than off-the-shelf API ones for the API Builder examples? Rent receipt is still a receipt no?

Do you have an idea on the one I should use? Maybe Train ticket. 

Rent-receipt is still a receipt but it's kinda different because it's specifically for rent and has different fields from the average receipt. In the off-the-shelf API  i saw only expense receipt.

File: build-your-first-document-parsing-api/training-interface.md [hunk 1]
```diff
@@ -271,4 +266,4 @@ When you call the predict endpoint with a trained model, it will perform an infe
 
  
 
-If you want to dive deeper into our product, feel free to reach out to our data scientist team: contact@mindee.com
\ No newline at end of file
+If you want to dive deeper into our product, feel free to reach out to our data scientist team: contact@mindee.com
```
[the remark this answers is not in the document]
Comment (fharper) [reply build-your-first-document-parsing-api/training-interface.md:263]: I asked the data science team

File: build-your-first-document-parsing-api/training-interface.md [hunk 2]
```diff
@@ -137,56 +133,52 @@ Click on any field input in the form, you should see the blue boxes on your docu
...
-
- 
-
-For example, let’s assume we are looking for the total amount of the invoice. 
-
+In the example above, we clicked on the Reservation Number field input in the form and the blue boxes highlight the Reservation Number field candidate in the document. When you hit the blue box corresponding to the Reservation Number in the document, the color changes and the Reservation Number is filled in the field.
  
 
-When you hit the blue box corresponding to the total amount in the document, the color changes and the amount is filled in the field.
+3. Next, click on **Validate**. When you “validate”, you are telling the deep learning model that this blue box is the one corresponding to the date. In other words, you are **teaching the model which candidate** is the date, and it’s going to learn to extract the right one on new samples.
```
Comment (fharper) [inline build-your-first-document-parsing-api/training-interface.md:141]: The screenshot below show reservation number as the paragraph on top, but here you mention it's corresponding to date.

File: build-your-first-document-parsing-api/training-interface.md [hunk 3]
```diff
@@ -137,56 +133,52 @@ Click on any field input in the form, you should see the blue boxes on your docu
...
-When you are going to press “validate”, you are going to tell the deep learning model that this green box was the one corresponding to the total amount. In other words, you are **teaching the model which candidate** is the total amount, and he’s going to learn to extract the right one on new samples.
+4. To send annotation for the model you have created, click on this **validate button** as seen below:
 
- 
+[block:image]
+{
+  "images": [
+    {
+      "image": [
+        "https://files.readme.io/7269372-Screenshot_2021-12-24_at_09.42.59.png",
```
Comment (fharper) [inline build-your-first-document-parsing-api/training-interface.md:168]: Maybe a screenshot with the mouse pointer over the validate button?

File: build-your-first-document-parsing-api/training-interface.md [hunk 4]
```diff
@@ -137,56 +133,52 @@ Click on any field input in the form, you should see the blue boxes on your docu
...
 At the very beginning, there is no trained model and calling the /predict endpoint will give you only the candidates for each field, no predictions.
 
- 
-
-Without any prediction, the /predict endpoint and the training cycle looks like this:
-
- 
-
- 
+ Without any prediction, the /predict endpoint and the training cycle looks like this:
```
Comment (fharper) [inline build-your-first-document-parsing-api/training-interface.md:181]: `/predict` in code formatting

File: build-your-first-document-parsing-api/training-interface.md [hunk 5]
```diff
@@ -205,9 +197,6 @@ Without any prediction, the /predict endpoint and the training cycle looks like
 }
 [/block]
  
-
- 
-
 The /annotations endpoint is automatically called within the interface, but you can also use the API if you want to do this on your own interface.
```
Comment (fharper) [inline build-your-first-document-parsing-api/training-interface.md:200]: `/annotations` in code formatting

**Review by fharper — CHANGES_REQUESTED** [review]
Some more modifications for this PR :)

File: build-your-first-document-parsing-api/training-interface.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fakela) [reply build-your-first-document-parsing-api/training-interface.md:263]: we can wait for a response from them by tomorrow, or we could add contact the devrel team(email address) if they are unsure or remove completely

File: build-your-first-document-parsing-api/training-interface.md (hunk 2 above)
Comment (fakela) [reply build-your-first-document-parsing-api/training-interface.md:141]: I missed that

File: build-your-first-document-parsing-api/training-interface.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fharper) [reply build-your-first-document-parsing-api/training-interface.md:263]: They didn't respond, I say remove it.

[the remark this answers is not in the document]
Comment (fharper) [reply build-your-first-document-parsing-api/training-interface.md]: I removed it

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**